usr/sbin/pkcsslotd/log.c

```diff
@@ -421,7 +420,8 @@ BOOL PKCS_Log(pLogHandle phLog, char *Format, va_list ap)
         }
     }
 
-    if (vsprintf(&(Buffer[0]), Format, ap) < 0) {
+    snres = vsnprintf(&(Buffer[0]), PATH_MAX, Format, ap);
```

Comment:
while you are at it, lets get rid of  this ugly `&(Buffer[0])`...
Why not just `Buffer` ?

Reply:
I'd also recomment to rename Buffer to buffer ... or buf and Formant to format or fmt ...

Reply:
Renamed to buf and fmt and fixed the name clash with the time buffer (called "buf" previously and "timebuf" now).